README.md

```diff
@@ -13,3 +13,15 @@ You can checkout and fork the Chart Authoring Tool on github:
 The Fluid community is an international group of designers, developers, and testers who focus on a common mission: improving the user experience and accessibility of the open web.
 
 The best way to join the Fluid Community is to jump into any of our community activities. Visit our [website](http://fluidproject.org/) for links to our mailing lists, chat room, wiki, etc.
+
+##Building
+
+Requirements:
+* `npm`
+
+Steps:
+* `npm install`
```

Comment:
The challenge there would be a completely "fresh" environment that doesn't yet have grunt installed for the project (so no `grunt install` until you've run `npm install`). That said, since it's only two steps, it would be simple enough to provide both a Windows (.bat) and a POSIX-compatible (Linux / OSX) script that runs both commands.